_posts/2015-12-14-05-Time-Series-Raster-In-R.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: post
```

Comment:
Thank you. That worked. Interesting that - this happened for this particular file but not all of the renamed files.